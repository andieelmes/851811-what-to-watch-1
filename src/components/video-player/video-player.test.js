import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from 'components/video-player/video-player.jsx';

const videoPlayerProps = {
  img: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  title: `Fantastic Beasts: The Crimes of Grindelwald`,
  active: true,
};

describe(`Video player component`, () => {
  it(`should render correctly`, () => {
    const tree = renderer
      .create(
          <VideoPlayer
            {...videoPlayerProps}
          />, {createNodeMock: (el) => el})
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
