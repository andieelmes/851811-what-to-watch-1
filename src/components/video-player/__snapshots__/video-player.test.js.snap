// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video player component should render correctly 1`] = `
<div
  className="small-movie-card__image"
  style={null}
>
  <img
    alt="Fantastic Beasts: The Crimes of Grindelwald"
    height="175"
    src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
    style={null}
    width="280"
  />
  <video
    controls={true}
    muted={true}
    poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
    style={
      Object {
        "display": "none",
      }
    }
  />
</div>
`;
